projetoFinal/include/tableau.h

```diff
@@ -0,0 +1,9 @@
+#include "jogo.h"
+
+#ifndef _TABLEAU_H_
+#define _TABLEAU_H_
+
+void renderizaCartasTableau(void *info, void *jogoVar);
```

Comment:
Essa função vai ser usada em outros arquivos? Pq se não for pode exportar somente a `renderizaTableau`